docs/modules/prometheus_ingester.md

```diff
@@ -60,23 +72,41 @@ Example of intended use case(s):
 ```
 
 #### type: `counter_increase`
-Supported result for this type of query: Matrix. [See Prometheus API documentation on details about these](https://prometheus.io/docs/prometheus/latest/querying/api/)
-Please note that the query is to be provided exactly as it would be passed to increase function, thus the following requirements applies:
-- **no range selector is allowed (e.g. `[5m]`)**. Range selector is automatically added by prometheus_ingester - for the first query it equals to configured `query.interval`, for all other, it equals to rounded difference of current timestamp and timestamp of the last sample (this should roughly to equal to `query.interval` as well).
 
-- **no sum()**. The query should result to matrix - list of time series. Do not apply sum function on the configured query for the same [reason why this is not a good idea to perform `increase(sum(...))`](https://www.robustperception.io/rate-then-sum-never-sum-then-rate).
+Supported result for this type of query:
+Matrix. [See Prometheus API documentation on details about these](https://prometheus.io/docs/prometheus/latest/querying/api/)
+Please note that the query is to be provided exactly as it would be passed to increase function, thus the following
+requirements applies:
+
+- **no range selector is allowed (e.g. `[5m]`)**. Range selector is automatically added by prometheus_ingester - for the
+  first query it equals to configured `query.interval`, for all other, it equals to rounded difference of current
+  timestamp and timestamp of the last sample (this should roughly to equal to `query.interval` as well).
+
+- **no sum()**. The query should result to matrix - list of time series. Do not apply sum function on the configured
+  query for the
+  same [reason why this is not a good idea to perform `increase(sum(...))`](https://www.robustperception.io/rate-then-sum-never-sum-then-rate)
+  .
 
 Mapping to resulting event(s):
 For every metric in the result:
-* If no previous sample for given metric is found, no event is generated and sample is just stored to local cache (please note that cache is not persisted upon restarts and is local to every individual instance of slo-exporter).
-* If previous sample of given metric is found, difference between new sample and previous sample is computed, and a single new event is generated with the following mapping:
+
+* If no previous sample for given metric is found, no event is generated and sample is just stored to local cache (
+  please note that cache is not persisted upon restarts and is local to every individual instance of slo-exporter).
+* Staleness applies same as with Prometheus. Last samples are dropped after the configured time, do not set the interval
+  longer than staleness otherwise you will be loosing data.  
```

Comment:
No, I believe this is right. Only if the data in cache is older than staleness threshold, it will be ignored.
So the query interval should be shorter than the staleness.